Add myself to members.md

Sibonji opened the pull request.

Sibonji committed:
Add myself to members.md

pavelkryukov commented on members.md:
```diff
@@ -7,3 +7,4 @@ List of 2021/2022 MIPT-V project members:
 6. Ivan Panferov
 7. Kirill Radkin
 8. Michael Bargatin
+9. Sibgatullin Bulat
```
Please follow same name order as peopel above

Sibonji committed:
Merge branch 'MIPT-ILab:main' into main

Sibonji committed:
Add myself to members.md

pavelkryukov commented:
We have not merged that because there is a merge conflict. Please resolve to proceed.

Sibonji committed:
Added member Otrashchenko Aleksey (#1502)

pavelkryukov committed:
Merge branch 'main' into main

pavelkryukov commented:
The PR is empty now.

Sibonji committed:
Byte-reverse (register) instruction
Mnemonic: rev8 rd, rs

Opcode: 1100100

This instruction reverses the order of the bytes in rs.

Operation:

let input = X(rs);
let output : xlenbits = 0;
letj = xlen - 1;

foreach (i from 0 to (xlen - 8) by 8) {
    output [i...(i + 7)] = input[(j - 7)...j];
    j = j - 8;
}

X[rd] = output;

Sibonji committed:
Merge branch 'MIPT-ILab:main' into main

Sibonji committed:
Changes to do pull

pavelkryukov commented on simulator/func_sim/alu.h:
```diff
@@ -236,6 +236,19 @@ struct ALU
+        size_t input = instr->v_src[0];
+        size_t output = 0;
+        for (size_t i = 0; i <= XLEN - 8; i += 8) {
+            (char) output [i/8] = (char) input [j/8];
```
This is not a valid C++ code…

Sibonji committed:
Merge branch 'MIPT-ILab:main' into main

Sibonji committed:
Merge branch 'MIPT-ILab:main' into main
